src/remote/activitypub/models/person.ts

```diff
@@ -31,6 +31,9 @@ import { normalizeForSearch } from '@/misc/normalize-for-search';
 
 const logger = apLogger;
 
+const nameLength = 128;
+const summaryLength = 2048;
```

Comment:
maybe these should be stored in `misskey/src/misc/hard-limits.ts` ?

Reply:
I will to do so!